core/math/color.cpp

```diff
@@ -415,11 +415,7 @@ Color Color::named(const String &p_name, const Color &p_default) {
 int Color::find_named_color(const String &p_name) {
 	String name = p_name;
 	// Normalize name.
-	name = name.replace(" ", "");
-	name = name.replace("-", "");
-	name = name.replace("_", "");
-	name = name.replace("'", "");
-	name = name.replace(".", "");
+	name = name.replace(Vector<String>({ " ", "-", "_", "'", "." }), "");
```

Comment:
Does it make sense to use static storage for these vectors to avoid the allocation overhead each time, such as
```suggestion
	static Vector<String> to_replace = { " ", "-", "_", "'", "." };
	name = name.replace(to_replace, "");
```

Reply:
Oh yes, probably!